static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -93,6 +93,9 @@ const initialise = (window) => {
                     },
                 },
             ],
+            bidCpmAdjustment : (bidCpm) => {
+                return bidCpm * 0.5;
```

Comment:
If I understand [the docs][] correctly, it should be 105% for a 5% boost:

```suggestion
                return bidCpm * 1.05;
```

[the docs]: https://docs.prebid.org/dev-docs/publisher-api-reference.html#22-bidcpmadjustment

Reply:
good catch, have changed it to 1.05